server/integration_tests/test_data/e2e_toolformer_rig_mode/numberofhouseholdsunderpovertylineinseattle/chart_config.json

```diff
@@ -0,0 +1,904 @@
+{
+  "client": "test_detect-and-fulfill",
+  "config": {
+    "categories": [
+      {
+        "blocks": [
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "statVarKey": [
+                      "Count_Household_HouseholderAge65OrMoreYears_FamilyHousehold_BelowPovertyLevelInThePast12Months"
```

Comment:
This is not a good match?

Reply:
I was using this as an example of skipping adding topics & unfortunately that's the highest score match .. https://screenshot.googleplex.com/AdTpyVqHPv48PDo

Reply:
actually this test is not needed anymore